### PR TITLE
#155965578  Set timeouts and grace values for days exceeding 30

### PR DESCRIPTION
**Pull request by betsybeth**
##### Login as a user: 
image
##### Click on grace period: 
image

### CODE REVIEW

**Review by lakivisi-zz — CHANGES_REQUESTED** [review]
Add tests for the above feature

**Comment by betsybeth** [comment]
Okay,Thanks.

**Comment by mutoromike** [comment]
Great implementation and understanding of the functionality.

File: hc/front/tests/test_update_timeout.py [hunk 1]
```diff
@@ -57,3 +57,22 @@ def test_it_checks_ownership(self):
         self.client.login(username="charlie@example.org", password="password")
         r = self.client.post(url, data=payload)
         assert r.status_code == 403
+        
+    def test_update_timeout_and_grace_works(self):
```
Comment (meshnesh) [inline hc/front/tests/test_update_timeout.py:61]: Good implementation of the test, although add a docstring

**Review by meshnesh — COMMENTED** [review]
You could refactor the code to practice DRY principles in the tests, most code has been rewritten over and over, they could be assigned a single variable, although good implementations

File: hc/front/tests/test_update_timeout.py (hunk 1 above)
Comment (betsybeth) [reply hc/front/tests/test_update_timeout.py:61]: Okay,Thanks

**Review by lakivisi-zz — APPROVED** [review]

**Pull request merged**